components/status/StatusPreviewCardMoreFromAuthor.vue

```diff
@@ -0,0 +1,19 @@
+<script setup lang="ts">
+import type { mastodon } from 'masto'
+
+defineProps<{
+  account: mastodon.v1.Account
+}>()
+</script>
+
+<template>
+  <div
+    max-h-2xl
+    flex gap-2
+    my-auto
+    bg-gray-300
+  >
+    <span z-0>More from</span>
```

Comment:
I added `z-0` here because the below `<AccountInlineInfo>` component covers the part of the text when hovering.